package.json

```diff
@@ -63,15 +64,16 @@
     "@ngrx/store": "^5.2.0",
     "cordova-ios": "4.5.5",
     "cordova-plugin-app-version": "0.1.9",
-    "cordova-plugin-compat": "1.2.0",
+    "cordova-plugin-app-preferences": "0.99.3",
+    "cordova-plugin-compat": "^1.2.0",
     "cordova-plugin-device": "^2.0.2",
-    "cordova-plugin-google-analytics": "1.8.6",
-    "cordova-plugin-inappbrowser": "3.0.0",
+    "cordova-plugin-google-analytics": "^1.8.6",
```

Comment:
can we revert these changes to versions?

Reply:
Done